functions.php

```diff
@@ -74,3 +74,61 @@ function error_message( $message ) {
 function trailingslashit( $string ) {
 	return rtrim( $string, '/' ) . '/';
 }
+
+/**
+ * Get WordPress version from a directory.
+ *
+ * @param  string $dir Path
+ * @return string
+ */
+function get_wordpress_version( $dir ) {
+	$wpVersion = file_get_contents( trailingslashit( $dir ) . 'src/wp-includes/version.php' );
```

Comment:
Don't we want the actual changeset reported with the WordPress version? While this WordPress version includes a changeset, it doesn't get incremented very often so it's likely to be incorrect.